.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Submit issues about Minetest Game
+    url: https://github.com/minetest/minetest_game/issues/new/choose
+    about: Only submit issues of the engine in this repository's issue tracker. Submit those of Minetest Game in its own issue tracker.
+  - name: Search for issue trackers of third-party subgames
```

Comment:
subgames -> games, might also wanna add something about looking for the issue tracker of mods